docs/docs/api_docs/pandas/io/read_sql_table.md

```diff
@@ -15,5 +15,7 @@
     in the warehouse (specified by `con`) where your table exists.
 -   Arguments `index_col`, `coerce_float`, `parse_dates`, `columns` and `chunksize` are
     not supported.
-
+-   Arguments `_snapshot_id` and `_snapshot_timestamp_ms` are only available for Iceberg tables. These Arguments
```

Comment:
Add examples since these arguments are not standard Pandas? Not clear what the data types are and how to get the values.